.github/workflows/code_style.yml

```diff
@@ -64,5 +64,29 @@ jobs:
           cmake ..
 
       - name: ShellCheck
-        run: make ie_shellcheck
+        run: cmake --build . --target ie_shellcheck -j8
+        working-directory: build
+
+  NamingConventionCheck:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Install Clang dependency
+        run: sudo apt --assume-yes install libusb-1.0-0-dev libclang-9-dev
```

Comment:
Do we really need to install `libusb-1.0-0-dev`?

Reply:
yes, for MYRIAD plugin. Alternative way is to disable plugin at all, but just copy pasted from the previous step